README.md

```diff
@@ -1,19 +1,113 @@
-# mattermost-plugin-jira ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-jira/master.svg) ![Codecov branch](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-jira/master.svg)
+# Mattermost Jira Plugin ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-jira/master.svg) ![Codecov branch](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-jira/master.svg)
```

Comment:
Jira or JIRA?

Reply:
`Jira`. Atlassian rebranded `JIRA` to `Jira`

Reply:
https://www.atlassian.com/legal/trademark I was under the wrong impression that the trademarked name is "JIRA", but evidently I was wrong and we should use "Jira"

Reply:
I am totally going to get rid of `JIRA` in the code now, will look so much better

Reply:
@levb All good. The old plugin code had `JIRA` because that's what they used before.